src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -174,26 +175,40 @@ def test_forecast_run_cmd():
     Tests that the command to be used to run the forecast executable was built successfully.
     """
     hera_expected = "srun --export=ALL test_exec.py"
-    assert hera_expected == FV3Forecast().run_cmd("--export=ALL", run_cmd="srun", exec_name="test_exec.py")
+    assert hera_expected == FV3Forecast().run_cmd(
```

Comment:
Looks like the linter may have gotten carried away here, I've had it do this to me. Double check your environment again and it should set it back

Reply:
Fixed the formatting issue - copy/paste the code from original.